app/views/kaui/payment_methods/_payment_methods_details_table.html.erb

```diff
@@ -1,16 +1,25 @@
 <table class="transparent">
+
   <tr>
-    <th>Name:</th>
+    <th><%= I18n.translate('name_with_colon') %></th>
```

Comment:
❤️ Thanks for doing these `I18n` updates 🙏 

We've had **many** requests over the years to provide translations of Kaui screens, we're slowly getting closer to being able to do it..